_component_design/buttons.md

```diff
@@ -368,3 +368,57 @@ Split drop-down buttons are preferred when more than one action is associated wi
 </div>
 
 </section>
+
+<section class="static-section" markdown="1">
+
+<div class="hxRow" markdown="1">
+
+{% column left:"hxCol-4 hxCol-xs-12 hxCol-sm-12 hxCol-md-4 hxCol-lg-4" %}
+
+### Download / export buttons
+
+- Download links **are always** formatted as: Download [noun] as [format]. 
+- Download is a word that focuses about the user's benefit because there is no ambiguity about the destination (the user's platform) and it will then be theirs.
+- Download links come in 4 layouts. The preferred layouts are the 1 and 2+ Download Options.
+- Use the condensed option in situations where space is too limited to use one of the preferred options.
```

Comment:
Which one is the condensed option? The 1-Download option?

Reply:
Condensed options are on the bottom. I'm going to redo the image here to make this more clear.

Reply:
I changed the image and referred to them as condensed variations to better isolate the two groups in the image; Text updated to below:

- Use a condensed button variation in situations where space is too limited to use one of the preferred options.
- The icon-only variation should be limited to use in a toolbar.